src/Components/LandingPage/LandingPage.tsx

```diff
@@ -56,7 +57,11 @@ export const LandingPage = () => {
   const [selectedBlueprint, setSelectedBlueprint] = useState<
     string | undefined
   >();
-  const { data: blueprints, isLoading } = useGetBlueprintsQuery({});
+  const [searchQuery, blueprintsFilter, setBlueprintsFilter] =
+    useDebouncedSearch('', 500);
+  const { data: blueprints, isLoading } = useGetBlueprintsQuery({
+    search: searchQuery,
```

Comment:
Can you please change the default search string to undefined? otherwise the [backend](https://github.com/osbuild/image-builder/blob/main/internal/v1/handler_blueprints.go#L191) triggers a redundant  regex search query.

Reply:
it is undefined - the debounce hook returns undefined if the filter is less then lenght (default 1).
But I agree it's not very straight forward approach.